sdk/web-pubsub/web-pubsub-express/test/connect.spec.ts

```diff
@@ -121,6 +230,36 @@ describe("Can handle connect event", function () {
     assert.equal(200, res.statusCode, "should be success");
   });
 
+  it("Should response with correct body when handler returns success for an mqtt request", async function () {
+    const endSpy = vi.spyOn(res, "end");
+    buildMqttRequest(req, "hub", "conn1", "physicalConnectionId");
+    const dispatcher = new CloudEventsDispatcher("hub", {
+      handleConnect: async (_, response) => {
+        response.success({
```

Comment:
how to you expect the user to use the connectionContext and know if it is ConnectionContext or MqttConnectionContext?